SPECS/kubernetes/kubernetes.spec

```diff
@@ -9,7 +9,7 @@
 %define container_image_components 'kube-proxy kube-apiserver kube-controller-manager kube-scheduler'
 Summary:        Microsoft Kubernetes
 Name:           kubernetes
-Version:        1.28.3
+Version:        1.28.4
```

Comment:
IMHO we are targeting the `fasttrack/2.0`, we should base our branch from `fasttrack/2.0` instead of `main`

Reply:
Changed the branch to main. fasttrack-2.0 is not up to date. I'm not sure how that branch is maintained.

Reply:
Please move the PR to `fasttrack/2.0` - @mfrw is right, this is a high severity CVE. And it works like any other branch, you need to apply the fix on top of it. It's meant to track `2.0` and have some additional fixes added to it for critical/high CVEs. Each fix in `fasttrack/2.0` then must be cherry-picked to `main`. It is the responsibility of the author of the original PR to perform the cherry-pick, but [here's a pipeline](https://dev.azure.com/mariner-org/mariner/_build?definitionId=2345) that triggers for each `fasttrack/2.0` merge, which should prepare that cherry-pick PR for you. You only need to make sure it's correct, resolve any (potential) merge conflicts, and merge it.

@0xba1a, you're right that it is possible, that `main` has other changes, but the way to resolve them, is to assume that whatever is on `fasttrack/2.0` came first, because these changes were already pushed to PMC and `main` wasn't.

Here's also [Jon's brownbag](https://microsoft-my.sharepoint.com/:v:/p/joslobo/EVnisfB6yrNBm3Ku4B1ksngBVolIVVQBm_n1JToy9ktblw) explaining the fast-track process in more detail.